src/WebJobs.Script/Description/ScriptFunctionInvokerBase.cs

```diff
@@ -139,9 +139,9 @@ protected virtual void OnScriptFileChanged(object sender, FileSystemEventArgs e)
         /// TEMP: We might find a better way to do this in the future, perhaps via core
         /// SDK changes.
         /// </summary>
-        protected static void ApplyAmbientBindingData(IBinder binder, IDictionary<string, string> bindingData)
+        protected static void ApplyAmbientBindingData(IBinderEx binder, IDictionary<string, string> bindingData)
```

Comment:
Does the C# binder need this call? 
If we switched the invokers to be more like C#, could we avoid this?